rpi-imager-haos.json

```
@@ -0,0 +1,26 @@
+{
+    "os_list": [
+        {
+            "name": "Home Assistant OS 5.13 (Pi 4 - 64-bit)",
+            "description": "Distributon optimized to run Home Assistant Core and Add-Ons.",
```

We should write it for people that don't know HA.

```suggestion
            "description": "Open-source home automation platform with a focus on local control and privacy.",
```